src/Misc/externals.sh

```diff
@@ -4,8 +4,10 @@ PRECACHE=$2
 
 NODE_URL=https://nodejs.org/dist
 UNOFFICIAL_NODE_URL=https://unofficial-builds.nodejs.org/download/release
-NODE12_VERSION="12.22.7"
-NODE16_VERSION="16.13.0"
+# https://github.com/nodejs/build/issues/2540
```

Comment:
To monitor: https://github.com/nodejs/build/issues/3077
Win-arm64 official build should be discussed in this meeting/issue

Reply:
FYI We're working towards it in https://github.com/nodejs/build/issues/3046#issuecomment-1315211429 but as always with a new platform it's dependent on people and systems to be able to elevate it to an official release, so for now pulling from the unofficial builds repo is the only way to get a versioned download from the nodejs project.

Reply:
Thanks for the heads-up @sxa , I somehow missed this one while digging up 😅